pygromos/files/blocks/imd_blocks.py

```diff
@@ -923,7 +923,7 @@ def adapt_energy_groups(self, energy_groups: Dict[int, int]):
             [description]
         """
         self.NEGR = len(energy_groups)
-        self.NRE = [energy_groups[last_atom] for last_atom in energy_groups]
+        self.NRE = [last_atom for last_atom in energy_groups]
```

Comment:
I don't recall anymore what is contained in energy groups, but it is a dict? 

so is the dictionary containing: {last_atom: bath}? does it make sense in this order???? because the initial line indicates it the other way around ({bath: last_atom})

Reply:
changed it to the intuitive version in commit 3f46fdb
https://github.com/rinikerlab/PyGromosTools/pull/105/commits/3f46fdb5af991a918ce04973c38864414f00f6cc